_posts/2022-11-07-multi-column-sorts-in-arrow-rust-part-2.md

```diff
@@ -212,7 +212,7 @@ A null value is encoded as a single `0x00` byte, and a non-null value encoded as
 
 ```
                           ┌─────┬─────┬─────┬─────┐
-   "Fabulous"             │ 01  │ 03  │ 05  │ 00  │
+   "Fabulous"             │ 01  │ 01  │ 02  │ 00  │
```

Comment:
Thank you -- I agree this is the correct value from the order preserving mapping

```
┌──────────┐                 ┌─────┐
│  "Bar"   │ ───────────────▶│ 01  │
└──────────┘                 └─────┘
┌──────────┐                 ┌─────┬─────┐
│"Fabulous"│ ───────────────▶│ 01  │ 02  │
└──────────┘                 └─────┴─────┘
┌──────────┐                 ┌─────┐
│  "Soup"  │ ───────────────▶│ 05  │
└──────────┘                 └─────┘
┌──────────┐                 ┌─────┐
│   "ZZ"   │ ───────────────▶│ 07  │
└──────────┘                 └─────┘
    Example Order Preserving Mapping
```